### PR TITLE
#683 add unit tests to reach full coverage

### PR DESCRIPTION
**Pull request by nick318**
The problem in #683:
small coverage of `TeeOutput` class.

The solution:
added few unit tests to reach full coverage.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#716](https://github.com/yegor256/cactoos/issues/716) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#716](https://github.com/yegor256/cactoos/issues/716) is assigned to @neonailol[/z](https://www.0crat.com/u/neonailol), here is [why](https://www.0crat.com/footprint/C63314D6Z/591746723). The budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4). Please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

**Comment by nick318** [comment]
@neonailol ping

**Comment by nick318** [comment]
@neonailol say @rultor  good to merge

**Comment by neonailol** [comment]
@rultor good to merge

**Comment by nick318** [comment]
@yegor256 ping

File: src/test/java/org/cactoos/io/TeeOutputTest.java [hunk 1]
```diff
@@ -41,16 +41,26 @@
  */
 public final class TeeOutputTest {
 
+    /**
+     * The CONTENT for copying of TeeOutput.
+     */
+    private static final String CONTENT = "Hello, товарищ!";
```
Comment (yegor256) [inline src/test/java/org/cactoos/io/TeeOutputTest.java]: @nick318 see http://www.yegor256.com/2016/05/03/test-methods-must-share-nothing.html

**Comment by yegor256** [comment]
@nick318 see above

**Comment by 0crat** [comment]
@neonailol[/z](https://www.0crat.com/u/neonailol) this job was assigned to you 5 days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by 0crat** [comment]
@neonailol[/z](https://www.0crat.com/u/neonailol) this job was assigned to you 8 days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by neonailol** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/yegor256/cactoos/issues/716#issuecomment-371760903))

@neonailol The user @neonailol[/z](https://www.0crat.com/u/neonailol) resigned from [#716](https://github.com/yegor256/cactoos/issues/716), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
Tasks refusal is discouraged, see [§6](http://www.zerocracy.com/policy.html#6): -15 points just awarded to @neonailol[/z](https://www.0crat.com/u/neonailol)

**Comment by 0crat** [comment]
This pull request [#716](https://github.com/yegor256/cactoos/issues/716) is assigned to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx), here is [why](https://www.0crat.com/footprint/C63314D6Z/680938845). The budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4). Please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

File: src/test/java/org/cactoos/io/TeeOutputTest.java (hunk 1 above)
Comment (nick318) [reply src/test/java/org/cactoos/io/TeeOutputTest.java]: @yegor256 removed shared content, take a look

**Comment by nick318** [comment]
@yegor256 take a look

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
@elenavolokhova[/z](https://www.0crat.com/u/elenavolokhova) please review this job completed by @fabriciofx[/z](https://www.0crat.com/u/fabriciofx), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#716](https://github.com/yegor256/cactoos/issues/716) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by elenavolokhova** [comment]
@0crat quality bad

**Comment by 0crat** [comment]
Quality is low, no payment, see [§31](http://www.zerocracy.com/policy.html#31)

**Comment by 0crat** [comment]
Quality review completed: +8 points just awarded to @elenavolokhova[/z](https://www.0crat.com/u/elenavolokhova)

**Pull request merged**